client/src/components/Header/header.css

```diff
@@ -121,3 +115,25 @@ li.user-state-link > a:focus {
 .ais-Hits {
   background-color: #fff;
 }
+
+@media (max-width: 630px) {
+  header {
+    height: 73px;
+    background-color: #87b787;
+  }
+
+  header .logo {
+    display: block;
+  }  
+
+  .fcc_searchBar {
+    padding: 3px 53px 0 10px;
+    position: absolute;
+    top: 36px;
+    width: 100%;
+  }
+
+  .app-wrapper {
```

Comment:
I don't think this class is available any more. You would perhaps need to apply these styles in the `Default` layout as all layouts use this as a base.